docs/apis/Alert.md

```diff
@@ -0,0 +1,39 @@
+# Alert
+
+`Alert` mirrors [React Native Alert Api](https://facebook.github.io/react-native/docs/alert.html).
+By default it'll look as ios alert, anyway there's possibility to style it. 
+## Methods
+
+static **setStyle**(config: Object)
```

Comment:
@necolas why don't you like `setStyle` API? the app can run, for example, on ios/web platforms in this case alert similar to android native alert will not make a lot of sense. I think it could be a good idea to export STYLE_ANDROID and STYLE_IOS. And keep possibility to switch to desired style.. BTW, android alerts has different styles from version to version, could you please point me to the image of alert you'd like to have?